servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcStatus.java

```diff
@@ -178,6 +179,13 @@ public String description() {
         return description;
     }
 
+    public GrpcStatus withDescription(String description) {
```

Comment:
We try to avoid adding new public API if it's not required. Consider using existing public ctor `GrpcStatus(GrpcStatusCode, null, description)` instead.

Reply:
sounds good, fixed.